pset-iterators.js

```diff
@@ -115,6 +131,12 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+const presidentsBetween1850and1900 = presidents.filter((president) => {
+  return parseInt(president.took_office) > 1849 && parseInt(president.took_office) < 1901
+})
+
+console.log(presidentsBetween1850and1900);
+
```

Comment:
@rlou88 Good job here using the filter method to return the presidents who served between 1850 and 1900. Good Job utilizing the parseInt method to turn the values of took_office  as a number.

Take a look at this alternate approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L120)